### DISCUSSION TITLE
background app refresh

### DISCUSSION THREAD

**Discussion by lodak80** [merlos/iOS-Open-GPX-Tracker#234] [Q&A] [answered]
Folks - Just wondering how this app collects navigation data without background app refresh? My phone has no cellphone signal and I still want to track all my movement during a day and then check it in the evening. Obviously the display will go off in 30 seconds on the phone and iphone will probably take the app out of memory

Thanks

**Comment by vincentneo** [comment, marked as answer]
This might help:

https://developer.apple.com/documentation/corelocation/getting_the_user_s_location/handling_location_events_in_the_background
> Core Location continues to deliver location events in the background, even if users disable the Background App Refresh setting for one or all apps.

**Comment by lodak80** [comment]
Wow so it sounds like you guys did the right thing in Xcode so everything
is set for me. Appreciate it!

**Comment by lodak80** [comment]
Sorry I'm also a little new to this. What's the best tool to export the gpx
files and will it have time stamps?

**Reply by merlos** [reply]
You can export the file with GPX tracker. Then you can open it in other apps in your computer such as  Avenue GPX Viewer https://github.com/vincentneo/Avenue-GPX-Viewer
